### PR TITLE
update interface body sharoom homepage

### PR DESCRIPTION
**Pull request by hvnien18it2**

### CODE REVIEW

File: src/features/ShareRoom/index.scss [hunk 1]
```diff
@@ -3,4 +3,37 @@
...
+.h{
+    margin-top: 30px;
+}
+.list_post{
+    border: groove;
+}
+
+
+.new{
+
```
Comment (NghiaBui486) [inline src/features/ShareRoom/index.scss]: Formating code

File: src/features/ShareRoom/page/list/index.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import './../../index.scss';
+
+const listData = [];
+for (let i = 0; i < 23; i++) {
+  listData.push({
+    href: 'http://ant.design',  //detail-title
+    name: `Tên bài đăng`,
+    avatar: 'https://bom.to/IKbv7z9cjfhCIt',
+    content: 'Diện tích - giá cả - địa điểm - người đăng - thời gian'
+
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/list/index.jsx]: Formating code

File: src/features/ShareRoom/page/list/index.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <List 
+    style={{paddingBottom:30}}
+    itemLayout="vertical"
+    size="large"
+    pagination={{
+      onChange: page => {
+        console.log(page);
+      },
+      pageSize: 5,    
+
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/list/index.jsx]: +1

File: src/features/ShareRoom/page/main/index.jsx [hunk 4]
```diff
@@ -1,13 +1,45 @@
 import React from "react";
-import Header from "../../../../components/Header";
-import Footer from "../../../../components/Footer";
+import { useHistory } from "react-router-dom";
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/main/index.jsx]: Why remove ';' this line

File: src/features/ShareRoom/page/main/index.jsx [hunk 5]
```diff
@@ -1,13 +1,45 @@
...
+    history.push("/user-login");
+  };
+
   return (
     <div className="wrapper-home">
-      <Header />
-      <div style={{ background: "white", height: 200 }}></div>
-      <Footer />
+      
+            <Header/>
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/main/index.jsx]: Formating code

File: src/features/ShareRoom/page/new/index.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    {
+      title: ' Title 5',
+    },
+  ];
+
+
+  class New extends React.Component {
+    render() {
+        return(
+
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/new/index.jsx]: Delete this line

File: src/features/ShareRoom/index.scss [hunk 7]
```diff
@@ -3,4 +3,37 @@
...
+}
+.list_post{
+    border: groove;
+}
+
+
+.new{
+   
+    border: groove;
+}
```
Comment (hvnien18it2) [inline src/features/ShareRoom/index.scss:39]: 123

File: src/features/ShareRoom/page/category/index.jsx [hunk 8]
```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Row, Col, Space } from 'antd';
+import TagsOutlined from '@ant-design/icons';
+import './../../index.scss';
+
+class Category extends React.Component{
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/category/index.jsx]: Using function component

File: src/features/ShareRoom/page/filter/index.jsx [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+  function handleChange(value) {
+   
+    console.log(`selected ${value}`);
+  }
+
+
+  
+
+  class Filter extends React.Component {
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/filter/index.jsx]: +1

File: src/features/ShareRoom/page/list/index.jsx [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+  listData.push({
+    href: 'http://ant.design',  //detail-title
+    name: `Tên bài đăng`,
+    avatar: 'https://bom.to/IKbv7z9cjfhCIt',
+    content: 'Diện tích - giá cả - địa điểm - người đăng - thời gian'
+
+  });
+}
+
+class List_post extends React.Component {
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/list/index.jsx]: +1

File: src/features/ShareRoom/page/new/index.jsx [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+    {
+      title: ' Title 4',
+    },
+    {
+      title: ' Title 5',
+    },
+  ];
+
+
+  class New extends React.Component {
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/new/index.jsx]: +1 using function component

File: src/features/ShareRoom/page/list/index.jsx (hunk 10 above)
Comment (NghiaBui486) [reply src/features/ShareRoom/page/list/index.jsx]: name must be ListPost or Posts?

File: src/features/ShareRoom/page/new/index.jsx [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+    render() {
+        return(
+
+            <div>
+            <Row>
+                <Col className='h'>
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/new/index.jsx]: why class name is 'h'?

File: src/features/ShareRoom/page/main/index.jsx [hunk 13]
```diff
@@ -1,13 +1,45 @@
...
+      <div style={{height:20}}></div>
+      
+      <div className='body'>
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/main/index.jsx]: can using class name wrapper-home__content replace for 'body'

File: src/features/ShareRoom/page/category/index.jsx [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+            </Row>
+            <div className='category'>
+            <Space direction="vertical">
+                <Row>
+                    <Space>
+                    <Col >
+                    <TagsOutlined style={{ fontSize: 20, color:'#8bb200' }} />
+                    </Col>
+                    <Col>
+                        <Link  className="name" to='/' style={{borderColor:'#8bb200'}} >Cho thuê phòng trọ</Link>
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/category/index.jsx]: can using category__item to replace for 'name'

Comment (NghiaBui486) [reply src/features/ShareRoom/page/category/index.jsx]: Research tag in antd and application for this component
here is e.g https://codepen.io/pen/?&editors=001

File: src/features/ShareRoom/page/filter/index.jsx [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+      render(){
+   // const [select_1, setSeLect_1] = useState("Loại phòng");
+    return (
+
+        <Row className="type_room" justify="space-between">
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/filter/index.jsx]: Change class name 'filter-room'. Note not using '_'

File: src/features/ShareRoom/page/filter/index.jsx [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+            <Space size='small'>
+            <Col span={4} className="btn_filter" >
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/filter/index.jsx]: +1 here 'filter-room__btn'

File: src/features/ShareRoom/page/filter/index.jsx [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+      render(){
+   // const [select_1, setSeLect_1] = useState("Loại phòng");
+    return (
+
+        <Row className="type_room" justify="space-between">
+            <Space size='small'>
+            <Col span={4} className="btn_filter" > 
+                <Button>Lọc <FilterOutlined /> </Button>
+            </Col>
+             <Col span={8}  className="select_item" >
```
Comment (NghiaBui486) [inline src/features/ShareRoom/page/filter/index.jsx]: +1 'filter-room__type'

**Pull request merged**